### PR TITLE
356 jetpack compose restore password mail sent

### PR DESCRIPTION
**Pull request by kerymov**

### CODE REVIEW

File: app/src/main/java/com/bogdan/codeforceswatcher/components/compose/AuthButton.kt [hunk 1]
```diff
@@ -15,13 +16,16 @@ import androidx.compose.ui.unit.dp
 fun AuthButton(
     label: String,
     modifier: Modifier = Modifier,
+    isInvert: Boolean = false,
```
Comment (yev-kanivets) [inline app/src/main/java/com/bogdan/codeforceswatcher/components/compose/AuthButton.kt:19]: `isInverted`?

File: app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        setContent {
+            AlgoismeTheme {
+                RestorePasswordMailSentScreen()
+            }
+        }
+    }
+
+    @Composable
+    private fun RestorePasswordMailSentScreen() {
+
```
Comment (yev-kanivets) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt:40]: Empty line?

File: app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                    AuthButton(getString(R.string.open_mail)) {
+                        startMailApp()
+                    }
+
+                    Spacer(Modifier.height(20.dp))
+
+                    AuthButton(
+                        label = getString(R.string.back_to_sign_in),
+                        modifier = Modifier.border(
+                            width = 2.dp,
```
Comment (yev-kanivets) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt:105]: Isn't it `2.dp` in other places too? Then you can embed it in the component. Maybe there are other similarities.

File: app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        try {
+            startActivity(Intent.createChooser(intent, "Email"))
+        } catch (e: ActivityNotFoundException) {
+            val toast = Toast.makeText(applicationContext, getString(R.string.mail_app_not_found), Toast.LENGTH_SHORT)
+            toast.show()
+        }
+    }
+
+    private fun startSignInActivity() {
+        startActivity(
```
Comment (yev-kanivets) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt:130]: Why not to one-line it?

File: app/src/main/java/com/bogdan/codeforceswatcher/features/auth/SignInComposeActivity.kt [hunk 5]
```diff
@@ -178,13 +178,13 @@ class SignInComposeActivity : ComponentActivity(), StoreSubscriber<AuthState> {
 
     private fun startSignUpActivity() {
         startActivity(
-            Intent(this@SignInComposeActivity, SignUpActivity::class.java)
```
Comment (yev-kanivets) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/auth/SignInComposeActivity.kt:181]: 👍

**Review by yev-kanivets — APPROVED** [review]
Good job 👍

File: app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt (hunk 3 above)
Comment (kerymov) [reply app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt:105]: This way we will have borders on all buttons. So, sometimes buttons will have the same background and border color. Is it ok?

Comment (kerymov) [reply app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt:105]: Also, some buttons have a border width of `1.6.dp`

Comment (yev-kanivets) [reply app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt:105]: Ah, OK, in this case never mind.

**Pull request merged**